src/interpolate/Interpolate_transform.re

```diff
@@ -0,0 +1,76 @@
+type cssTransform = {
+  transform: string,
+  transformProperty: Js.Nullable.t(string),
+};
+
+[@bs.deriving jsConverter]
+type transformProperties = [
+  | `translate
+  | `translateX
+  | `translateY
+  | `translateZ
+  | `skew
+  | `skewX
+  | `skewY
+  | `skewZ
+  | `rotate
+  | `rotateX
+  | `rotateY
+  | `rotateZ
+  | `scale
+  | `scaleX
+  | `scaleY
+  | `scaleZ
+  | `perspective
+];
+
+let parseTransform = transform => {
+  switch (String.index(transform, '(')) {
+  | idx =>
+    /* Decompose the translate string into the translate property, i.e. translateX, skew, etc.
+       Then, extract the value, i.e. 20px, 50deg, so we can begin to interpolate it. */
+    let property =
+      Js.String.substring(~from=0, ~to_=idx, transform)
+      ->transformPropertiesFromJs;
+    let value =
+      Js.String.substring(
+        ~from=idx + 1,
+        ~to_=String.index_from(transform, idx + 1, ')'),
+        transform,
+      );
+
+    switch (property) {
+    | Some(p) => {
+        transform: value,
+        transformProperty: Js.Nullable.return(p->transformPropertiesToJs),
+      }
+    | None => {transform: value, transformProperty: Js.Nullable.null}
+    };
+  | exception Not_found => {transform, transformProperty: Js.Nullable.null}
```

Comment:
This is how you handle `exception`s in Reason. If we don't find an opening parenthesis in the string handed us by the user, we'll return `null` for the `transformProperty`. This will cause our `getInterpolatorForPair` function to say, "Alright, I bet this isn't a valid CSS transform."